Mathlib/Topology/Algebra/Field.lean

```diff
@@ -32,8 +32,8 @@ theorem Filter.tendsto_cocompact_mul_right₀ [ContinuousMul K] {a : K} (ha : a
     Filter.Tendsto (fun x : K => x * a) (Filter.cocompact K) (Filter.cocompact K) :=
   Filter.tendsto_cocompact_mul_right (mul_inv_cancel₀ ha)
 
-/-- Compact Hausdorff topological fields are finite. -/
-instance (priority := 100) {K} [DivisionRing K] [TopologicalSpace K]
+/-- Compact Hausdorff topological fields are finite. This is not an instance to avoid slowdowns. -/
```

Comment:
Can you be more specific than "slow-down"? "Slowly failing typeclass searches" is already better,
but noting this instance would apply to every `Finite` goal would be best.
Same for the other comment.